components/Dropdown/Dropdown.js

```diff
@@ -0,0 +1,35 @@
+// function myFunction() {
+//     document.getElementById('myDropdown').classList.toggle('show')
+// }
+// let button = document.querySelector('.Dropdown__button')
+// button.onclick = function(event) {
+//         let dropdowns = document.getElementById('myDropdown')
+//         console.log(dropdowns)
+//         dropdowns.classList.toggle('show')
+// }
+
+//Refactoring my JS to match what Dan did
+
+class Dropdown {
```

Comment:
Hm, I know that you mentioned that you refactored to match Dan but the dropdown seems to be finicky. I was able to click on it and it would provide me the menu but it wasn't in dropdown form. I'm not quite sure if this was on purpose so I'll leave you to it! 😄